README.md

```diff
@@ -56,9 +56,16 @@ Your lab directory must include
 ## BONUS 1pt
 * create a second module `readManyFiles` that can read an array of any length of file paths and return an array of coresponding text data
  * rewrite index.js to use `readManyFiles`
- 
+
 ## Rubric  
 * 2ps Configuration
 * 3pts Feature Tasks
 * 3pts Tests
 * 2pts Documentation
+
+## Using Done In callbacks
```

Comment:
Great overview on Mocha's ```done```! Very good explanation. 

For future README's, go ahead and move your README writings to the very top so it's easier to find. You can even delete all of Duncan's contents in the README so you can write your own stuff. :)